packages/ui/src/models/visualization/flows/pipe-visual-entity.ts

```diff
@@ -48,6 +60,10 @@ export class PipeVisualEntity implements BaseVisualCamelEntity {
   getNodeLabel(path?: string): string {
     if (!path) return '';
 
+    if (path === ROOT_PATH) {
+      return this.id;
```

Comment:
shouldn't rather this be the name of the `Pipe` and fallback to the `id` if not present?

Reply:
We are always assigning the pipe name as `id` so will there be any chance of `id` being undefined?
As far as I think, pipe name can be update either from the source-code or from the form, both of the cases are handled!

 Case 1. [When the pipe name gets updated form the Source-code](https://github.com/shivamG640/kaoto/blob/Fix_1031/packages/ui/src/models/visualization/flows/pipe-visual-entity.ts#L39).
 Case 2.  [When the pipe name gets updated from the form](https://github.com/shivamG640/kaoto/blob/Fix_1031/packages/ui/src/models/visualization/flows/pipe-visual-entity.ts#L105)

Reply:
You're right @shivamG640 :+1: , let's keep it as it is